api/add_compression_policy.md

```diff
@@ -24,6 +26,12 @@ the [integer_now_func][set_integer_now_func] to be set).
 |---|---|---|
 | `if_not_exists` | BOOLEAN | Setting to true causes the command to fail with a warning instead of an error if a compression policy already exists on the hypertable. Defaults to false.|
 
+>:WARNING: 
+Compression policies on continuous aggregates should be set up so that they do
+not overlap with refresh policies on continuous aggregates. This is due to a 
+current TimescaleDB limitation that prevents refresh of compressed regions of
+continuous aggregates.
```

Comment:
```suggestion
<highlight type="important">
Compression policies on continuous aggregates should be set up so that they do
not overlap with refresh policies on continuous aggregates. This is due to a 
current TimescaleDB limitation that prevents refresh of compressed regions of
continuous aggregates.
</highlight>
```